.github/actions/report-job-status/action.yml

```diff
@@ -0,0 +1,45 @@
+name: 'Send VK Teams message on failure'
```

Comment:
It should be the same as in the README.md file.

Reply:
Done